tlslite/keyexchange.py

```diff
@@ -274,6 +275,9 @@ def processClientKeyExchange(self, clientKeyExchange):
             raise TLSLocalAlert(AlertDescription.illegal_parameter,
                                 "Invalid dh_Yc value")
 
+        if dh_Yc % self.dh_p == 0:
+            raise TLSIllegalParameterException("Suspicious dh_Yc value")
```

Comment:
Just add new unit test to `unit_tests/test_tlslite_keyexchange.py` to class `TestDHE_RSAKeyExchange`, yes? 
If yes, can be that condition before `if not 2 <= dh_Yc <= self.dh_p - 1:` condition? Because if I want to fulfill condition, I need `dh_Yc` >= `dh_p`, but that condition, that I mentioned, doesnt allow it.
And last question - what about values `dh_Yc` and `dh_p` in unit test? Can I use some constants like 3 and 6 or what values should I use?